boards/xtensa/esp32s2/common/scripts/.gitignore

```
@@ -0,0 +1 @@
+/*.ld.tmp
```

```suggestion
/*.ld.tmp

```
Missing newline at the end of file

---

Fixed.